### ISSUE TITLE
HA 2023.5.0 - State values no longer show the friendly name

### ISSUE THREAD

**Issue by clearwave1** [custom-cards/button-card#693]
**Checklist**
- [x ] I updated the card to the latest version available
- [ x] I cleared the cache of my browser

**Describe the bug**
After upgrading to HA 2023.5.0 the state values in the cards are now in lowercase and are the actual state values and not the state value "friendly name". E.g. It used to show "Disarmed Away" and now shows "disarmed_away". Other Lovelace cards show the state properly.

**Version of the card**
Version: v3.5.0

**To Reproduce**
This is the configuration I used:
```yaml
type: custom:button-card
entity: person.don
name: Don
show_state: true
```

**Screenshots**
![image](https://user-images.githubusercontent.com/53064625/236248118-d318b4ac-3f20-42bd-bd86-a5a599f54072.png)

**Expected behavior**
It should work as it did before HA version 2023.5.0 where the friendly name for the state was displayed. In the attached picture you will notice the standard Entity Card shows "Home" with a capital 'H' and the Button Card shows "home".

**Desktop (please complete the following information):**
 - Browser - Chrome or Edge
 - Version - latest

**Comment by RomRider** [comment]
I'll review that over the weekend. I've seen a PR about that too.

**Comment by sstefanowski** [comment]
I guess it's just a duplicate of #685. The "home" is just the raw state name while "Home" is the official translation of this state to English ;)
We all are waiting for the approval of the fix in pull request here:
https://github.com/custom-cards/button-card/pull/692
Please @RomRider can you accept this PR quickly?

**Comment by RomRider** [comment]
I'll do so tomorrow, not in front of a computer at the moment

**Comment by piesinopoli** [comment]
Any news?

**Comment by clearwave1** [comment]
Any update on this one? Even a workaround would be appreciated.

**Comment by skynet01** [comment]
Plus one on this one. Shows not_home for me instead of Away :)

**Comment by Laho812** [comment]
+1
![Screenshot_20230612-012650_Home Assistant](https://github.com/custom-cards/button-card/assets/119891233/82207669-cac0-4d93-95da-50e72b9ee53a)

For the Lights (Leselichter) the states should show "An" and "Aus" instead of "On" and "Off".
For the contact sensors (Türen und Fenster) the states should show "Offen" and "Geschlossen" instead of "On" and "Off".

A fix would be nice cause I love the custom button card and I want the translation back, because it then looks much more professional...

**Comment by Ricks88** [comment]
@RomRider 
Any new updates about this?
All has changes to on/off instead of translations.
![244930094-6bbfd46f-1dd1-43f1-89c2-c919f44da991](https://github.com/custom-cards/button-card/assets/91467918/52019d79-8779-42f7-814c-95fa90af2f4f)
![244930078-393ab1d5-353c-4c44-b41a-5eec569edcea](https://github.com/custom-cards/button-card/assets/91467918/0f30a27e-39fc-40cb-9cb6-d685b373ce49)

**Comment by pablitosd** [comment]
I just have to. "Door is off/on" .I had open and closed. :(

**Comment by RomRider** [comment]
Duplicate of #685, closing in favor of the other one.
It's released in version beta 4.0.0-dev1 if you want to test it out.
